test/regression/cases/ddot_metrics/experiment.yaml

```diff
@@ -4,12 +4,11 @@ erratic: false
 target:
   name: datadog-agent
   cpu_allotment: 4
-  memory_allotment: 2GiB
+  memory_allotment: 7.5 GiB
```

Comment:
too big for current limits, will not run successfully

Reply:
Interesting, it does run. I thought the target had 8GiB available.

Reply:
I double checked this and wrote it down in our team docs. We do have enough memory to handle this allotment. (https://datadoghq.atlassian.net/wiki/spaces/SMP/pages/5277978466/Resource+Limits)

Reply:
oops, thanks George, good timing on that